package.json

```diff
@@ -11,6 +11,7 @@
   },
   "exports": {
     ".": "./src/index.js",
+    "./hmr": "./src/hmr/module.js",
```

Comment:
Doesn't need to be exposed or exported, it'd be a part of the core modules that auto load, similar to the root `builder`